app/src/androidTest/java/com/github/polybooks/FillSaleTest.kt

```diff
@@ -0,0 +1,77 @@
+package com.github.polybooks
+
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.action.ViewActions.scrollTo
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.matcher.IntentMatchers
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import org.hamcrest.Matchers
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+class FillSaleTest {
+
+    @get:Rule
+    val activityRule = ActivityScenarioRule(FillSaleActivity::class.java)
+
```

Comment:
I won't do this in this PR since it isn't it's purpous. Do another PR

Reply:
I mean, it's a class you created from scratch for this PR, so I feel it is related to this PR, no? 
But fine, now that you've already merged it, it can be done in another PR.